ContactDetailsApi/V2/Gateways/PersonDbGateway.cs

```diff
@@ -33,8 +33,21 @@ public async Task<IEnumerable<Person>> GetPersons(List<Guid> ids)
                 batchGet.AddKey(id);
             }
 
+            Person SafeToDomain(PersonDbEntity person)
+            {
+                try
+                {
+                    return person?.ToDomain();
+                }
+                catch (Exception e)
+                {
+                    _logger.LogError(e, "Error: Failed to convert person {PersonDb} to Person domain", person?.Id);
+                    return null;
+                };
+            };
+
```

Comment:
Same as with Safe Tenure conversion.

Reply:
same as above - it allows us to easily know in the logs which record is causing the issue

Reply:
I see, so it's a prophylactic measure.